docs/guides/send-logs-from-vector-to-quickwit.mdx

```diff
@@ -0,0 +1,198 @@
+
+<Tabs>
+
+<TabItem value="docker" label="Docker">
+
+```bash
+curl -o otel-logs.yaml https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/otel-logs/index-config.yaml
+# Create quickwit datadir.
+mkdir -p qwdata
+# Create index.
+docker run -v $(pwd)/qwdata:/quickwit/qwdata -v $(pwd)/otel-logs.yaml:/quickwit/index-config.yaml quickwit/quickwit index create --index-config /quickwit/index-config.yaml
+```
+
+Then we start Quickwit server that will be ready so receive logs on `/api/v1/otel-logs/ingest` endpoint:
+```bash
+docker run --init --rm -v $(pwd)/qwdata:/quickwit/qwdata -p 127.0.0.1:7280:7280 quickwit/quickwit run
+```
+
+</TabItem>
+
+<TabItem value="cli" label="CLI">
+
+```bash
+curl -o otel-logs.yaml https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/otel-logs/index-config.yaml
+# Create quickwit datadir.
+mkdir -p qwdata
+# Create index.
+./quickwit index create --index-confg otel-logs.yaml
+```
+
+Then we start Quickwit server that will be ready so receive logs on `/api/v1/otel-logs/ingest` endpoint:
+
+```bash
+./quickwit run
+```
+
+</TabItem>
+
+</Tabs>
+
+
+## Setup Vector
+
+Vector is well-known for collecting logs from every parts of your infrastructure, transform and aggregate them, and finally forward them to a sink.
+Our sink here will be Quickwit ingest API `http://127.0.0.1:7280/api/v1/otel-logs/ingest`.
+To keep it simple in this tutorial, we will use a log source called `demo_logs` that generates logs in a given format. Let's choose the common `syslog` format
+(Vector does not generate logs in the OTEL format directly!) and use the transform feature to map the `syslog` format into the OpenTelemetry format.  
+
+
+```toml title=vector.toml
+[sources.generate_syslog]
+type = "demo_logs"
+format = "syslog"
+count = 100
+
+[transforms.remap_syslog]
+inputs = [ "generate_syslog"]
+type = "remap"
+source = '''
+  structured = parse_syslog!(.message)
+  .timestamp, err = to_unix_timestamp(structured.timestamp)
+  .body = .message
+  del(.message)
+  .resource.source_type = .source_type
+  .resource.host.hostname = structured.hostname
+  .resource.service.name = structured.appname
+  .attributes.syslog.procid = structured.procid
+  .attributes.syslog.facility = structured.facility
+  .attributes.syslog.version = structured.version
+  del(.source_type)
+  .severity = if includes(["emerg", "err", "crit", "alert"], structured.severity) {
+    "ERROR"
+  } else if structured.severity == "warning" {
+    "WARN"
+  } else if structured.severity == "debug" {
+    "DEBUG"
+  } else if includes(["info", "notice"], structured.severity) {
+    "INFO"
+  } else {
+   structured.severity
+  }
+  .name = structured.msgid
+'''
+
+[sinks.emit_syslog]
+inputs = ["remap_syslog"]
+type = "console"
+encoding.codec = "json"
+
+[sinks.quickwit_logs]
+type = "http"
+inputs = ["remap_syslog"]
+encoding.codec = "ndjson"
+uri = "http://0.0.0.0:7280/api/v1/otel-logs/bulk"
```

Comment:
I struggled a bit on this ip thingy, look up my comment

Reply:
the IP is wrong here, should be on 127.0.0.1.